### ISSUE TITLE
"Temporary allow" firewall notification settings

### ISSUE THREAD

**Issue by ameshkov** [AdguardTeam/AdguardForAndroid#421]
Original request:
http://forum.xda-developers.com/showpost.php?p=66153599&postcount=939

> Dont know if it has been added but it would be nice if the for firewall blocked app notification "Would you like to temporarly enable firewall for X app" to only apply for specific user selected apps. This becouse I have several apps that I dont want a notification as mentioned above for.

**Comment by evgeniy-ADG** [comment]
>  I currently have my settings with mobile data blocked by default so I can manually allow or block on demand app requests, but it would be great if you could add all 3 access options (allow, allow temporarily, or block) directly to notification pop-up.

**Comment by ameshkov** [comment]
Merged into https://github.com/AdguardTeam/AdguardForAndroid/issues/3379
